content/docs/0.8.1/deploy/upgrade/longhorn-manager.md

```diff
@@ -82,6 +82,21 @@ If the migration fails and the error log mentioned above is printed out, users n
 
 3. Use the Longhorn UI to recreate the PV and PVC.
 
+4. Due to removing the compatible csi deployment, there might be the following error message in kubelet logs:
+    ```
+    clientconn.go:1120] grpc: addrConn.createTransport failed to connect to {/var/lib/kubelet/plugins/io.rancher.longhorn-reg.sock 0  <nil>}. Err :connection error: desc = "transport: Error while dialing dial unix /var/lib/kubelet/plugins/io.rancher.longhorn-reg.sock: connect: connection refused". Reconnecting...
+    ```
+   And it can be fixed by removing the `io.rancher.longhorn-reg.sock` from the kubelet on the node with the following command:
+
+>**Note**: Please make sure there is no PV running with driver `io.rancher.longhorn`:
```

Comment:
The format is messed up starting from this line.

Reply:
Each new line inside of a step should be indented four spaces.

Reply:
@yasker , @catherineluse , sorry, which line are you referring to? And what's wrong with it?
Thanks.

Reply:
You can see it in the rich diff or the preview page.

Reply:
like this

```
4. Due to removing the compatible CSI deployment, there might be the following error message in the kubelet logs:
    ```
    clientconn.go:1120] grpc: addrConn.createTransport failed to connect to {/var/lib/kubelet/plugins/io.rancher.longhorn-reg.sock 0  <nil>}. Err :connection error: desc = "transport: Error while dialing dial unix /var/lib/kubelet/plugins/io.rancher.longhorn-reg.sock: connect: connection refused". Reconnecting...
    ```
   And it can be fixed by removing the `io.rancher.longhorn-reg.sock` from the kubelet on the node with the following command:

    >**Note**: Please make sure there is no PV running with driver `io.rancher.longhorn`.

    ```
    /var/lib/kubelet/plugins_registry/io.rancher.longhorn-reg.sock
    ```
    Meanwhile, the kubelet will log the following message:
    ```
    plugin_watcher.go:212] Removing socket path /var/lib/kubelet/plugins_registry/io.rancher.longhorn-reg.sock from desired state cache
    ```
```

Reply:
@catherineluse , thanks. Just re-submitted the fix each line within a step has intended 4 spaces.